cli/start.go

```diff
@@ -155,9 +156,6 @@ func runStart(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	info := util.GetBuildInfo()
-	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
```

Comment:
it was kind of nice having the build info in stderr as well. Without it, you need to match stdout (which has no timestamp) with startup lines in stderr.